Tests/UKHO.MaritimeSafetyInformation.Web.UnitTests/Services/UserServiceTests.cs

```diff
@@ -0,0 +1,71 @@
+﻿using FakeItEasy;
+using Microsoft.AspNetCore.Http;
+using NUnit.Framework;
+using System.Security.Claims;
+using UKHO.MaritimeSafetyInformation.Web.Services;
+
+namespace UKHO.MaritimeSafetyInformation.Web.UnitTests.Services
+{
+    [TestFixture]
+    public class UserServiceTests
+    {
+        private const string DISTRIBUTOR_ROLE_NAME = "TBC";
```

Comment:
Since this is the Unit Test case and also, distributor role value (derived from B2C token) is going to be same for distributor through out the application ,so we have declared it as private constant variable.